ooniprobe/View/TestResults/LogViewController.m

```diff
@@ -44,6 +66,47 @@ - (void)viewDidLoad {
     self.textView.scrollEnabled = true;
 }
 
+/*
+ *allora l’unica cosa che suggerisco e’ di documentare cosa fa la funzione
+ dicendo che se non parsa restituisce la stringa stessa
+*/
```

Comment:
I believe we should probably not merge this comment to master.

Reply:
LOL, maybe not even here